README.md

```diff
@@ -7,260 +7,26 @@
 
 Java client for the [kubernetes](http://kubernetes.io/) API.
 
-## Client versioning
-The Java client uses Semantic Versioning. We increment the major version number whenever we
-regenerate the client for a new Kubernetes release version (see table below). Whenever we do
-this there are new APIs added and possibly breaking changes in the generated Kubernetes API
-Stubs. Whenever you upgrade a major version, be prepared for potential breaking changes.
+## To start using Kubernetes Java Client
 
-## Installation
+See the wiki page and documentation [here](https://github.com/kubernetes-client/java/wiki).
 
-To install the Java client library to your local Maven repository, simply execute:
+- [Installation](https://github.com/kubernetes-client/java/wiki/1.-Installation)
+- [Client Versioning and Compatibility](https://github.com/kubernetes-client/java/wiki/2.-Versioning-and-Compatibility)
+- [Code Examples](https://github.com/kubernetes-client/java/wiki/3.-Code-Examples)
 
-```shell
-git clone --recursive https://github.com/kubernetes-client/java
-cd java
-mvn install
-```
+## Development
 
-Refer to the [official documentation](https://maven.apache.org/plugins/maven-deploy-plugin/usage.html) for more information.
+- [Development and Contributing](https://github.com/kubernetes-client/java/wiki/4.-Development-and-Contributing)
+- [Generate Java CRD Models](https://github.com/kubernetes-client/java/wiki/5.-Generate-Java-CRD-Model)
+- [Known Issues](https://github.com/kubernetes-client/java/wiki/6.-Known-Issues)
 
-### Maven users
+## Support
 
-Add this dependency to your project's POM:
+If you need support, start with checking whether you'r hitting known issues. If that doesn't work, please open an issue
+to describe the cases. Additionally, before you file an issue, please search existing issues to see if your issue is 
```

Comment:
linkify 'existing issues' to point to the issue list?

Reply:
linkified